docs/whatsnew_1.5.7.html

```diff
@@ -173,16 +172,17 @@ <h2>
                 The most notable improvements and bugfixes in v1.5.7 since v1.5.6 include:
             </h2>
             <ul>
-                <li>Core - Updated hundreds of files for compatibility with PHP 7.2, 7.3 and 7.4.  <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
-                <li>Admin - Default notification behavior for order update (send email/do not send email/hide update) may now be set</li>
-                <li>Admin - Admins can now login to customer accounts to place orders</li>
-                <li>Admin - Added product-details popover to allow quick visibility into products bought in an order without opening the order</li>
-                <li>Admin - Validation for integer min and max values and email addresses</li>
-                <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
-                <li>Admin - Order status updatable from order listing screen</li>
-                <li>Admin - Improve many configuration setting descriptions</li>
-                <li>Admin - On-page help added, with links to new documentation</li>
-                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Core - Many files have been updated for compatibility with PHP 7.2, 7.3 and 7.4. <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
+                <li>Admin - Added a sort order to the Order Statuses so the admin order update dropdown can mirror your business workflow.</li>
+				<li>Admin - The default notification behavior for an Order Status update (send email/do not send email/hide update) may now set by an admin switch.</li>
+                <li>Admin - Order Status may be updated from the order listing screen.</li>
+				<li>Admin - Order Status emails are now sent in the language of the order.</li>
+				<li>Admin - Admins can now login to a customer account (login as that customer), to place an order on their behalf.</li>
```

Comment:
Please fix alignment - it's hard to compare otherwise.  I thought this line had been removed.

Reply:
ok